plugin/core/views.py

```diff
@@ -766,22 +766,17 @@ def run(self, view_id: int, command: str, args: Optional[Dict[str, Any]] = None)
 <style>
     html {{padding: 0; background-color: transparent}}
 
-    #lsp-color-box {{
+    #lsp-color-box a {{
+        border: 1px solid color(var(--foreground) alpha(0.25));
+        background-color: {backgroundColor};
         margin-top: 0.1em;
         height: 1rem;
         width: 1rem;
-        border: 1px solid color(var(--foreground) alpha(0.25));
-        background-color: {backgroundColor};
-    }}
-
-    #lsp-color-box a {{
-        font-size: 1.7rem; /** HACK - make the font bigger to make the whole color box clickable. */
-        text-decoration: none;
-        color: #00000001;
+        display: block;
     }}
 </style>
 <body id='lsp-color-box'>
-    <a href="{command}">▬</a>
+    <a href="{command}"><img/></a>
```

Comment:
`img` is not a self-closing element so technically `<img>` is more correct.
But also I doubt that `<img>` is really necessary here. Probably `&nbsp;` would be enough. Or frankly nothing should work too, if minihtml works the same as HTML here.